tools/docker-compose-datasets-server.yml

```
@@ -197,7 +197,7 @@ services:
       CONFIG_NAMES_MAX_NUMBER: ${CONFIG_NAMES_MAX_NUMBER-3_000}
       DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY: ${DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY-/stats-cache}
       DESCRIPTIVE_STATISTICS_HISTOGRAM_NUM_BINS: ${DESCRIPTIVE_STATISTICS_HISTOGRAM_NUM_BINS-10}
-      DESCRIPTIVE_STATISTICS_MAX_PARQUET_SIZE_BYTES: ${DESCRIPTIVE_STATISTICS_MAX_PARQUET_SIZE_BYTES-100_000_000}
+      DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES: ${DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES-5_000_000_000}
```

let's keep the same value? Or does it help to change it?

```suggestion
      DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES: ${DESCRIPTIVE_STATISTICS_MAX_DATASET_SIZE_BYTES-100_000_000}
```